### ISSUE TITLE
Bug (BULKRAX): Create and Validate

### ISSUE THREAD

**Issue by orangewolf** [scientist-softserv/louisville-hyku#86]
# Summary
User receives an error message when selecting "create and validate" on a new bulkrax csv importer. 

The error is valid because "create and validate" does not import anything. Therefore, when the code checks for the collection's id, it is correct that it does not exist. 

The code should not be checking for the existence of a collection at this point, when "create and validate" is chosen. 

# Acceptance Criteria

* [ ] selecting "create and validate" on a csv bulkrax importer should not throw an error.

# Screenshots or Video

URL (Staging): https://lv-hyku-staging.notch8.cloud/importers/26?locale=en#collection-entries

DEMO: https://share.getcloudapp.com/RBun27ZP

clicking create and validate would cause this error:

![Image](https://user-images.githubusercontent.com/1054448/204091222-14112937-458f-465f-b9e3-2ef22c2d7588.png)

# Testing Instructions

* sample file: [rhino.zip](/uploads/1cc55667c571361aefe93a03f9e715b4/rhino.zip)
* Create and Validate the sample file using a csv importer - confirm there are no errors
* click continue to finish importer 

## BULKRAX REFERENCE
The work of this ticket will be done in bulkrax. Ref: https://github.com/samvera-labs/bulkrax/issues/495

Once that ticket is merged into main, the task of this story is to pull in the changes with: 

`bundle update bulkrax --conservative`

And verify the fix on staging.

**Comment by orangewolf** [comment]
Shana May 2022
---
PR: https://github.com/samvera-labs/bulkrax/pull/506
TODO: update project's bulkrax version /gemfile.lock once PR is approved in bulkrax

**Comment by orangewolf** [comment]
Selam May 2022
---
# Notch8 QA
**Passes** testing criteria. 
- [ ] sample file: [rhino.zip](/notch8/louisville-hyku/uploads/1cc55667c571361aefe93a03f9e715b4/rhino.zip)
- [ ] Create and Validate the sample file using a csv importer - confirm there are no errors
- [ ] click continue to finish importer

**Steps:**
- Created a new import and selected create and validate 
- Redirected me to the importer show page which showed me the correct number of works and collections according to the attached CSV to the sample zip, 'rhino' - NO ERRORS mentioned
- selected continue and was redirected to the index page with a success banner notification.
- Checked works and collection for correct alignment with CSV. The rhino zip had 3 works, two of which are children of one parent collection. Both works, collections, and relationships were correct. 

Reference screen recording to view a successful "Create and Validate"
https://drive.google.com/file/d/1YJ1LZtAr6RdD8dSyGa2Ld7wkwWJooHMK/view?usp=sharing



![Image](https://user-images.githubusercontent.com/1054448/204091274-1fdf62a2-0047-420d-8138-9e1f7c43aad2.png)

**Comment by orangewolf** [comment]
Rachel May 2022
---
I couldn't watch the video (don't have permission) but I tried to test this on staging myself. See illuminated manuscript collection. No errors but the images aren't showing in the viewer or thumbnails.

**Comment by orangewolf** [comment]
Alisha May 2022
---
we discussed this during the client meeting today.

the way that the "create and validate" option had been working *was the bug*. the expectation for that button is to only create the importer and validate the csv. it shouldn't create any objects/records at all.

"create and import" will actually import the object.

**Comment by orangewolf** [comment]
Rachel May 2022
---
OK, I went back and was able to import the files for the validated CSV by updating the importer.
